docs/README.md

```diff
@@ -28,173 +25,263 @@ Github Authenticator is supported by Identity Server 5.1.0 upwards.
+7. Under **Federated Authentication**, select the identity provider you created from the drop-down.
 
-The next step is to configure the service provider.
-Learn how to [Adding and Configuring a Service Provider](https://docs.wso2.com/display/IS570/Adding+and+Configuring+a+Service+Provider)
+   <img src="images/GithubServiceProvider.png" alt="GithubServiceProvider.pn" /> 
+   Ensure that the **Federated Authentication** radio button is selected 
+8. Click  **Update** to save the changes. 
 
-1. Return to the management console.
+You have now successfully added and configured the service provider.
 
-2. In the `Service Providers` section, click `Add` under the `Main` tab.
+## Testing the Sample
 
-3. Since you are using travelocity as the sample, enter travelocity.com in the `Service Provider Name` text box and click `Register` .
+Follow the steps below to test the sample application.
 
-4. In the `Inbound Authentication Configuration` section, click `Configure` under the `SAML2 Web SSO Configuration` section.
+1. Go to the following URL: `http://<TOMCAT_HOST>:<TOMCAT_PORT>/travelocity.com/index.jsp`, e.g., `http://localhost:8080/travelocity.com`. 
+   Note that depending on the tag that you used to checkout sample, the travelocity home page would defer.  
 
-5. Now set the configuration as follows:
-    1. Issuer: travelocity.com
-    2. Assertion Consumer URL: http://localhost:8080/travelocity.com/home.jsp
+    1. If you checked out from a tag prior to v5.7.0, select **Click here to login with SAML from WSO2 Identity Server**.
     
-6. Select the following check-boxes:
-    1. Enable Response Signing.
-    2. Enable Single Logout. 
-    3. Enable Attribute Profile.
-    4. Include Attributes in the Response Always.
-
-   ![5](images/Travelocity-Service-Provider.png "Travelocity-Service-Provider.png")
+       <img src="images/Travelocity.jpeg" alt="Travelocity home screen from a tag prior to v5.7.0" width="500"/>
    
-7. Click `Update` to save the changes. Now you will be sent back to the `Service Providers` page.
-
-8. Navigate to the Local and `Outbound Authentication Configuration` section.
- 
-9. Select the identity provider you created from the drop-down list under `Federated Authentication`.
-![6](images/GithubServiceProvider.png "GithubServiceProvider.png")
-
-10. Ensure that the `Federated Authentication` radio button is selected and click  `Update` to save the changes. 
-
-You have now added and configured the service provider.
-
-## Testing the sample
-
-1. To test the sample, go to the following URL: http://<TOMCAT_HOST>:<TOMCAT_PORT>/travelocity.com/index.jsp. E.g., http://localhost:8080/travelocity.com
-
-2. Login with SAML from the WSO2 Identity Server.
-![7](images/Travelocity.jpeg "Travelocity.jpeg")
+    2. If you checked out from the tag v5.7.0, select **Click here to login with SAML (Redirect biding) from WSO2 Identity Server*
```

Comment:
formatting problem, should be **Click here to login with SAML (Redirect biding) from WSO2 Identity Server**